### ISSUE TITLE
Question about acitpo theme: How can I change the width of posts and pages? 

### ISSUE THREAD

**Issue by LOOLinc** [rogerhub/acitpo#1]
Hi Roger!

First of all I want to say thanks for making and sharing the awesome theme!
I really like, though, I I'm trying to make a few adjustments to suit  my needs.  

I can do some very basic coding but wasn't able to do the things I wanted, so I wondered if you could give me some hints? 
- I want to make the size of both posts and pages wider (I think they are around 500 pixels wide currently). 
  - Preferably with pages being much wider than posts

So basically, how/where do I set minimum and maximum size for posts and pages? 

Thanks

Cheers,
Nils Sørensen

**Comment by rogerhub** [comment]
Hi there, this theme uses SASS to preprocess the CSS. If you look in the sass/_variables.sass, you can change `$post_width` to whatever number you want, and then re-compile the CSS with `sass sass/style.sass style.sass`. You need to do the recompiling step, or else the changes won't show up. Let me know if you need assistance.

By the way, you can always change the width of a single page/post by adding inline CSS inside the post. Just play around with the styles in Web Inspector until you're satisfied, and copy the CSS to a `<style type="text/css"></style>` block inside your post content.

**Comment by LOOLinc** [comment]
Thank you for your answer.

I am afraid you need to flesh it out a bit more for me to understand.  
I can't seam to find the sass/_variables.sass you mention, in which line is it (and is it in the stylesheet.css)?
Do you have a link to a guide on how to recompile the CSS? 
  Sorry, this is new to me :)

About using the inline CSS: It is a really good idea to use for pages, since I have very few pages! I have installed the the CSS Plus plugin and put in: `.page {  width: 900px;}` , which makes the page wider, but floating to the right, and mobile devices does not really seam to register the full page width. So I think I need to include some of your code, but I am not sure what of it? 

Thanks again!

**Comment by rogerhub** [comment]
Nevermind, let me backtrack. I think all you need to do is add `body.page .main { width: 900px; }` to your custom CSS plugin.

**Comment by LOOLinc** [comment]
Thanks for you help dude! This code did the trick for me:  
`.main {
  margin: 50px auto 0 auto;
  max-width: 970px;
  min-width: 610px;
  padding: 1px 20px; }`

However, It looks [quite weird on mobile devices](lolink.dk). Even though I set a minimum width for a post or page, an mobile browser will still take the old with as default. 
I have a wide banner/logo and must of that is not rendered on a mobile browser.
Any ideas on what I should look at? 

It's cool if you don't want to bother with me mutilating your beautiful theme btw ;)

**Comment by rogerhub** [comment]
Can you post a screenshot?

**Comment by LOOLinc** [comment]
Here is a [screenshot](https://dl.dropboxusercontent.com/u/2840929/Screenshot_2015-01-21-00-31-03.png). If you have a mobile device, you could also visit [this site](http://lolinc.dk) and try to scroll to the left.
